mapserver/osm.map.mako

```diff
@@ -111,6 +107,127 @@ LAYER
 END
 % endfor
 
+# "column": "tourism",
+# "value": "zoo",
+# "type": "zoo",
+# "name": u"Zoos"
+LAYER
+    NAME "tourism_activity"
+    GROUP "zoo" #compatibily
+    EXTENT 473743 74095 839000 306400
+    TYPE POINT
+    STATUS ON
+    TEMPLATE fooOnlyForWMSGetFeatureInfo # For GetFeatureInfo
+    CONNECTIONTYPE postgis
+    PROCESSING "CLOSE_CONNECTION=DEFER" # For performance
+    CONNECTION "user=${dbuser} password=${dbpassword} host=${dbhost} dbname=osm"
+    DATA "geom FROM (SELECT regexp_replace(format(\'%s\', name), \'^$\', osm_id::text) AS display_name,name,osm_id,access,aerialway,amenity,barrier,bicycle,brand,building,covered,denomination,ele,foot,highway,layer,leisure,man_made,motorcar,\"natural\", operator, population, power, place, railway, ref, religion, shop, sport, surface, tourism, waterway, wood, way AS geom FROM planet_osm_point) AS foo USING UNIQUE osm_id USING srid=21781"
+    PROCESSING "NATIVE_FILTER=tourism in ('zoo','information','museum','viewpoint')"
+    LABELITEM "name"
+    PROJECTION
+        "init=epsg:21781"
+    END
+    TOLERANCE 10
+    TOLERANCEUNITS pixels
+    CLUSTER
+       MAXDISTANCE 40  # in pixels
+       REGION "ellipse"  # can be rectangle or ellipse
+       GROUP ('[tourism]' = 'information') # a logical expression to specify the grouping condition
+     END
```

Comment:
you want to add the cluster setup to the template LAYER object? This is out of the scope of this PR and I am not sure we want this for all layers from the template. We will discuss this wednesday ;)